lib/nerves_hub_link/configurators/nerves_key.ex

```diff
@@ -0,0 +1,54 @@
+defmodule NervesHubLink.Configurator.NervesKey do
```

Comment:
We should wrap the defmodule here in `Code.ensure_loaded?(:nerves_key)` to prevent the module from being compiled with bad x_refs if `nerves_key` is not loaded.

Reply:
Instead of ignoring bad xrefs in `mix.exs`?

Reply:
Done